src/main/java/com/kobylynskyi/graphql/codegen/GraphQLCodegen.java

```diff
@@ -43,403 +43,406 @@
-                if (!Utils.isEmpty(fieldDefinition.getInputValueDefinitions())) {
-                    Map<String, Object> fieldProjDataModel = dataModelMapperFactory.getRequestResponseDefinitionMapper().mapParametrizedInput(mappingContext, fieldDefinition, definition);
-                    generatedFiles.add(GraphQLCodegenFileCreator.generateFile(mappingContext, FreeMarkerTemplateType.PARAMETRIZED_INPUT, fieldProjDataModel, outputDir));
-                }
-            }
-        }
-        return generatedFiles;
-    }
-
-    private Optional<File> generateFieldResolver(MappingContext mappingContext,
-                                                 List<ExtendedFieldDefinition> fieldDefinitions,
-                                                 ExtendedDefinition<?, ?> parentDefinition) {
-        if (Boolean.TRUE.equals(mappingConfig.getGenerateApis())) {
-            List<ExtendedFieldDefinition> fieldDefsWithResolvers = fieldDefinitions.stream()
-                    .filter(fieldDef -> FieldDefinitionToParameterMapper.generateResolversForField(mappingContext, fieldDef, parentDefinition))
-                    .collect(toList());
-            if (!fieldDefsWithResolvers.isEmpty()) {
-                Map<String, Object> dataModel = dataModelMapperFactory.getFieldDefinitionsToResolverMapper().mapToTypeResolver(mappingContext, fieldDefsWithResolvers, parentDefinition.getName());
-                return Optional.of(GraphQLCodegenFileCreator.generateFile(mappingContext, FreeMarkerTemplateType.OPERATIONS, dataModel, outputDir));
-            }
-        }
-        return Optional.empty();
-    }
-
-    private File generateInput(MappingContext mappingContext, ExtendedInputObjectTypeDefinition definition) {
-        Map<String, Object> dataModel = dataModelMapperFactory.getInputDefinitionMapper().map(mappingContext, definition);
-        return GraphQLCodegenFileCreator.generateFile(mappingContext, FreeMarkerTemplateType.TYPE, dataModel, outputDir);
-    }
-
-    private File generateEnum(MappingContext mappingContext, ExtendedEnumTypeDefinition definition) {
-        Map<String, Object> dataModel = dataModelMapperFactory.getEnumDefinitionMapper().map(mappingContext, definition);
-        return GraphQLCodegenFileCreator.generateFile(mappingContext, FreeMarkerTemplateType.ENUM, dataModel, outputDir);
-    }
-
-    protected void initCustomTypeMappings(Collection<ExtendedScalarTypeDefinition> scalarTypeDefinitions) {
-        for (ExtendedScalarTypeDefinition definition : scalarTypeDefinitions) {
-            if (definition.getDefinition() != null) {
-                mappingConfig.putCustomTypeMappingIfAbsent(definition.getDefinition().getName(), "String");
-            }
-            for (ScalarTypeExtensionDefinition extension : definition.getExtensions()) {
-                mappingConfig.putCustomTypeMappingIfAbsent(extension.getName(), "String");
-            }
-        }
-    }
+	protected final MappingConfig mappingConfig;
```

Comment:
In this repository, we use spaces instead of tabs, sorry 😄
Could you please change only the required lines and use spaces?
Thanks